src/Reader/StandaloneExtensionManager.php

```diff
@@ -9,9 +9,9 @@
 
 namespace Zend\Feed\Reader;
 
-class StandaloneExtensionManager implements ExtensionManagerInterface
+final class StandaloneExtensionManager implements ExtensionManagerInterface
```

Comment:
We can't make it final, it will require a major version bump as someone is quite likely already extending it.